benchmark/distributed/master.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+
+const jstp = require('../..');
+
+const yargs = require('yargs');
+const fs = require('fs');
+const path = require('path');
+const parseAddress = require('./parse-address');
+const statistics = require('../statistics');
+
+const args = yargs
+  .option('address', {
+    alias: 'A',
+    describe: 'Address for master server to listen on',
+    type: 'string',
+    default: 'tcp://localhost:3000',
+  })
+  .options('report', {
+    alias: 'R',
+    describe: 'Report interval in ms',
+    type: 'number',
+    default: 10000,
+  })
+  .options('key', {
+    alias: 'K',
+    describe: 'path to file that contains SSL certificate key',
```

Comment:
First letter style in `describe` should be consistent, either all fields have the first letter uppercased or all don't.